cmd/dumpling/main.go

```diff
@@ -56,8 +57,12 @@ func init() {
 	flag.Uint64Var(&fileSize, "F", export.UnspecifiedSize, "The approximate size of output file")
 	flag.Uint64Var(&fileSize, "filesize", export.UnspecifiedSize, "The approximate size of output file")
 
-	flag.StringVar(&outputDir, "output", ".", "Output directory")
-	flag.StringVar(&outputDir, "o", ".", "Output directory")
+	flag.StringVar(&outputDir, "output", defaultOutputDir(), "Output directory")
+	flag.StringVar(&outputDir, "o", defaultOutputDir(), "Output directory")
```

Comment:
Save the `defaultOutputDir()` into a variable so that both `-o` and `--output` are using the same default value.